File: Assignment_16/Q3.c
///////////////////////////////////////////////////////
//
//  File name :     assignment16-Q3.c
//  Descreption :   accept no of rows and column from user and display below pattern
//  Author :        Vaishanvi D Shingare
//  Date :          28/04/2025

// input : iRow = 5     iCol = 5
/*  output -  $       *       *       *       *
              #       $       *       *       *
              #       #       $       *       *
              #       #       #       $       *
              #       #       #       #       $
*/             
//
///////////////////////////////////////////////////////



 #include<stdio.h>

void Pattern(int iRow , int iCol)
{
    int i = 0, j = 0;

    for(i = 1; i<= iRow; i++)
    {
        for(j = 1; j <= iCol; j++)
        {
            if((i >=j ) && (i <= j) == 0)
            {
                printf("#\t");
            }
            else if(i == j)
            {
                printf("$\t");

            }
            else
            {
                printf("*\t");
            }
            
        }
        printf("\n");
    }



}

int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("enter numbers of rows and columns :");
    scanf("%d %d",&iValue1,&iValue2);

    Pattern(iValue1,iValue2);

    return 0;
}

File: Assignment_46/Q3.java
///////////////////////////////////////////////////////////////////////////////////////////////
//
//  File name :     assignment46-Q3.java
//  Descreption :   write java program which accepts nos of rows and colunm from user and display below patterns.
//  Author :        Vaishanvi D Shingare
//  Date :          28/01/2025


// Input 3  5   
/*
     A       A       A       A       A
     B       B       B       B       B
     C       C       C       C       C
*/

import java.util.*;

class Pattern
{
    public void Display(int iRow, int iCol)
    {
        int i = 0, j = 0;
        char ch = 'A';

        for(i = 1; i <= iRow; i++,ch++)
        {
            for(j = 1; j <= iCol; j++)
            {
                System.out.print(ch+"\t");
            }
            System.out.print("\n");
        }
    }
}

class Q3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue1 = 0, iValue2 = 0;

        System.out.println("Enter number of rows : ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter number of columns : ");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue1, iValue2);
    }
}